tests/smoke.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+
+const APP_ID = process.env.SMOKE_TEST_APP_ID || '';
+const VERSION = process.env.NPM_PACKAGE_VERSION || '';
```

Comment:
Is this an NPM package version? Seems like it's the WFE version 🤔

Reply:
We're using this environment variable in the CI, which contains the WFE version. It's derived from the `package.json` file, so I think this is why the name was chosen.

Reply:
interesting, thx for the context!